From 42aff7a7bd5b00c1f49f266188040b5f3f0cb7c5 Mon Sep 17 00:00:00 2001
From: Xie Haocheng <haocheng.xie@amd.com>
Date: Fri, 4 Nov 2022 16:37:34 +0800
Subject: [PATCH 0362/1646] [Backport] add CONFIG_PERF_EVENTS_AMD_BRS=y to
 openeuler_defconfig

amd inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5ZIMJ

Reference: https://gitee.com/openeuler/kernel/commit/022afa9d6d3d1ae04344dc5f7bdbfaa9a211efb9

--------------------------------

Enable AMD perf branch sample feauture in x86 openeuler defconfig.
It's implemented as a bool type option, must set 'y' to enable it.

Signed-off-by: Xie Haocheng <haocheng.xie@amd.com>
Signed-off-by: Lin Yujun <linyujun809@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 1840007c9776..945cfded56ae 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -401,6 +401,7 @@ CONFIG_PERF_EVENTS_INTEL_UNCORE=m
 CONFIG_PERF_EVENTS_INTEL_RAPL=m
 CONFIG_PERF_EVENTS_INTEL_CSTATE=m
 CONFIG_PERF_EVENTS_AMD_POWER=m
+CONFIG_PERF_EVENTS_AMD_BRS=y
 # end of Performance monitoring
 
 CONFIG_X86_16BIT=y
-- 
2.17.1

